include/cling/Interpreter/RuntimePrintValue.h

```diff
@@ -15,6 +15,7 @@
 #endif
 
 #include <string>
+#include <vector>
```

Comment:
I'd be very much in favor of removing the #include <string> by allowing the caller to provide the storage. Including string brought significant % of STL, so including vector might be cheap because of that.

@marsupial did you check what is the runtime cost in terms of memory and performance of adding these #include.